pallets/collator-selection/src/lib.rs

```diff
@@ -331,14 +331,35 @@ pub mod pallet {
 			T::PotId::get().into_account()
 		}
 
+		pub fn try_remove_candidate(who: &T::AccountId) -> Result<usize, DispatchError> {
```

Comment:
pub items should ideally be documented, specially for this case where it is mutating state.